addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -310,7 +310,13 @@ var SnippetEditor = Widget.extend({
             });
         });
 
-        this.trigger_up('go_to_parent', {$snippet: this.$target});
+        const parent = this.$target[0].parentElement;
+        const nextSibling = this.$target[0].nextElementSibling;
+        const previousSibling = this.$target[0].previousElementSibling;
+        this.trigger_up('activate_snippet', {
+            $snippet: $(previousSibling || nextSibling || parent)
+        });
```

Comment:
Seems way nicer and shorter without jQuery :+1: Just: I think you can remove all the code related to the go_to_parent event right? I don't think it will be used anymore?